js_modules/dagit/packages/core/src/assets/LaunchAssetExecutionButton.tsx

```diff
@@ -58,70 +61,102 @@ export const LaunchAssetExecutionButton: React.FC<{
       disabledReason || 'Assets must be in the same job to be materialized together.';
   }
 
+  const assetsWithConfig = assets.filter((a) => configSchemaForAssetNode(a));
+  const anyAssetsHaveConfig = assetsWithConfig.length > 0;
+
+  if (anyAssetsHaveConfig && partitionDefinition) {
+    disabledReason =
+      disabledReason || 'Cannot materialize assets using both asset config and partitions';
+  }
+
   title = title || 'Refresh';
-  if (partitionDefinition) {
+
+  let tooltipChildren: React.ReactNode;
+  if (anyAssetsHaveConfig) {
+    const assetOpNames = assets.map(({opNames}) => (opNames as string[])[0]);
+    const sessionPresets = {
+      solidSelection: assetOpNames,
+      solidSelectionQuery: assetOpNames.map((name) => `"${name}"`).join(' '),
+    };
+    const jobName = '__ASSET_GROUP';
```

Comment:
Hmm I don't think this is guaranteed to be the job name - the asset group is just a prefix now and not the name of the job. Can you use `jobName` from up above instead of redefining it here?